wordpress-mobile/WordPress-iOS
# Login: updated magic links design

Refs #7272

Implements the new design for the magic links flow. Check that illustration! ❤️ 

Without a Gravatar | With a Gravatar | Open Email
--------|--------|--------
![magic-links-without-gravatar](https://user-images.githubusercontent.com/517257/27417658-45f59b52-56d3-11e7-8c7e-21ff445602c5.png)|![magic-links-with-gravatar](https://user-images.githubusercontent.com/517257/27417662-49d186a0-56d3-11e7-9a46-1221e339d5d3.png)|![magic-links-email](https://user-images.githubusercontent.com/517257/27417667-4fa16c4e-56d3-11e7-9bed-670ceda79505.png)

To test:
- inspect the appearance for correctness
- verify that magic link flow hasn't been broken ;)

Needs review: @aerych
